DotNetWebApi/Controllers/LocationController.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DotNetWebApi.Models;
+
+namespace DotNetWebApi.Controllers;
+
+[Route("api/")]
```

Comment:
Best practices have a base route set in Program.cs using app.UsePathBase so all API routes will be prefixed with api.

Then this Route attribute can be changed to Route("[controller"]) to automatically pickup the rout name. In this example the controller would still be at api/picnic but it would not be specified on each endpoint.